model/labels/labels_common.go

```diff
@@ -46,7 +46,7 @@ func (ls Labels) String() string {
 	ls.Range(func(l Label) {
 		if i > 0 {
 			b.WriteByte(',')
-			b.WriteByte(' ')
+			// b.WriteByte(' ')
```

Comment:
Only two tests (see below) have failed when removing this. (+ `tsdb dump` output will change, I'll add a test for it to avoid regressions)

Apart of `tsdb dump`, I don't think we really rely on those spaces, it'd even reduce confusion by making the format close to the text scrape ones.

If you don't think we should change `tsdb dump`, what do you think about the implementation below (commented)?